src/libraries/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs

```diff
@@ -941,6 +943,7 @@ public sealed class SocketsHttpHandler_SchSendAuxRecordHttpTest : SchSendAuxReco
         public SocketsHttpHandler_SchSendAuxRecordHttpTest(ITestOutputHelper output) : base(output) { }
     }
 
+    [SkipOnMono("Tests hang with chrome. To be investigated", TestPlatforms.Browser)]
```

Comment:
do we have a tracking issue for this?

Reply:
Should any of the SocketsHttpHandler tests be working?

Reply:
No, SocketsHttpHandler should throw PNSE on wasm-browser

Reply:
Ok, let's follow up on this.